src/DataFixtures/ORM/LoadVotingPlatformElectionData.php

```diff
@@ -224,8 +224,12 @@ private function getCandidates(): array
 
         /** @var CommitteeCandidacy $committeeCandidacy */
         $committeeCandidacy = $this->getReference('committee-candidacy-1');
-        $candidate1->setImagePath($committeeCandidacy->getImagePath());
-        $candidate2->setImagePath($committeeCandidacy->getImagePath());
+
+        array_map(function (Candidate $candidate) use ($committeeCandidacy) {
+            if (rand(0, 1)) {
```

Comment:
to make sure we'll have both cases we could check something like `0 === $count++ % 2` instead of random bool

Reply:
good idea, I will change it in the next PR 👍